crane/views/crane.py

```diff
@@ -0,0 +1,31 @@
+"""
+Non-public view for use by admins to see a list of repositories served by crane.
+"""
+from flask import Blueprint, current_app, json, render_template, request
+
+from .. import app_util
```

Comment:
We don't have an official entrypoint for our testing environments, but I wish we did. Then we wouldn't have to use `..` as a relative import. There is no better way to do this that I know of yet.